core/src/main/java/com/rudderstack/core/internal/ConfigDownloadServiceImpl.kt

```diff
@@ -87,12 +93,27 @@ internal class ConfigDownloadServiceImpl @JvmOverloads constructor(
 
     }
 
+    override fun addListener(listener: ConfigDownloadService.Listener, replay: Int) {
+        val replayAccepted = replay.coerceAtLeast(0)
+        for (i in (downloadSequence.size - replayAccepted).coerceAtLeast(0) until downloadSequence.size) {
+            listener.onDownloaded(downloadSequence[i])
+        }
```

Comment:
I have a question. Under what circumstances will `listener.onDownloaded(downloadSequence[i])` be called? Because when I tested it, it wasn't called.

Reply:
Can check